tool/teleport/common/teleport.go

```diff
@@ -495,6 +499,12 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	integrationConfExternalAuditCmd.Flag("glue-table", "The name of the Glue table used.").Required().StringVar(&ccf.IntegrationConfExternalAuditStorageArguments.GlueTable)
 	integrationConfExternalAuditCmd.Flag("aws-partition", "AWS partition (default: aws).").Default("aws").StringVar(&ccf.IntegrationConfExternalAuditStorageArguments.Partition)
 
+	// cloud-aws-cred is handled in enterprise code
+	if options.CloudAWSCredCmdArg != nil {
+		cloudAWSCredCmd := app.Command("cloud-aws-cred", "Helper command used by Teleport Cloud to produce credentials.").Hidden()
+		cloudAWSCredCmd.Arg("file", "Credential file to output").Required().StringVar(options.CloudAWSCredCmdArg)
```

Comment:
This command allows an attacker with command execution to exfiltrate files. We should mitigate this by restricting which files it can access. An even saner implementation would use a non-standard path so we don't expose our enterprise users creds.

Reply:
Will remove the Subcommand arg in favor of hardcoding the path. My next question would be, do you think I should still conditionally configure this Cmd based on enterprise binary or not? I no longer have need for the `*string` run option but could use a `bool` to add it when configured by enterprise?

Reply:
> do you think I should still conditionally configure this Cmd based on enterprise binary or not

Yes -- mostly because we don't want to litter up OSS with hidden commands, even if they don't do anything.  The conditional seems like a small cost to pay.